.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: Report a bug in Quarkus
+labels: kind/bug
+body:
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: >-
+        Describe the issue you are experiencing here to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+
+        Provide a clear and concise description of what the problem is.
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: >-
+        Describe the expected behavior clearly and concisely.
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      description: >-
+        Describe the actual behavior clearly and concisely.
+  - type: textarea
+    attributes:
+      label: How to Reproduce?
+      description: >-
+        Link to a small reproducer (preferably a Maven project if the issue is not Gradle-specific) or attach an archive containing the reproducer to the issue.
+      placeholder: |
+        Steps to reproduce the behavior:
```

Comment:
Can you add `Reproducer: ` before the steps with a new line? We have so many bug reports without reproducer, I would like to make it very explicit.

Reply:
That's a valid concern. Should we make this field mandatory?